generate.sh

```diff
@@ -35,3 +35,35 @@ if grep -qFx 'namespace {' "$FILE"; then
 else
     printf '\n/**\n * WordPress database abstraction object.\n * @var wpdb\n */\n$wpdb = \\null;\n' >>"$FILE"
 fi
+
+# Add ReturnTypeWillChange attribute to PHP 8-incompatible methods.
+declare -r -a REQUESTS_V1_METHODS=(
+    'Requests_Utility_FilteredIterator::unserialize'
+    'Requests_Utility_FilteredIterator::__unserialize'
+    'Requests_Utility_FilteredIterator::current'
+    'Requests_Cookie_Jar::offsetExists'
+    'Requests_Cookie_Jar::offsetGet'
+    'Requests_Cookie_Jar::offsetSet'
+    'Requests_Cookie_Jar::offsetUnset'
+    'Requests_Cookie_Jar::getIterator'
+    'Requests_Utility_CaseInsensitiveDictionary::offsetExists'
+    'Requests_Utility_CaseInsensitiveDictionary::offsetGet'
+    'Requests_Utility_CaseInsensitiveDictionary::offsetSet'
+    'Requests_Utility_CaseInsensitiveDictionary::offsetUnset'
+    'Requests_Utility_CaseInsensitiveDictionary::getIterator'
+)
+for METHOD in "${REQUESTS_V1_METHODS[@]}"; do
+    # Get the line number where the method is defined.
+    LINE="$(php -r "require 'wordpress-stubs.php'; print (new ReflectionMethod('${METHOD}'))->getStartLine();")"
+    echo "${METHOD} is defined on line ${LINE}."
```

Comment:
@janw-me I've added a check here. Some methods does not exists in earlier core releases.